docs/www/rpk-commands.md

```diff
@@ -222,13 +222,13 @@ Aliases:
   list, ls
 ```
 
-### topic set-config ![linux icon][linux] ![mac icon][mac]
+### topic alter-config ![linux icon][linux] ![mac icon][mac]
```

Comment:
This change should be covered in the release notes for the release when it was changed.